storage/olddbclean/oldDatabaseCleaner.go

```diff
@@ -0,0 +1,231 @@
+package olddbclean
+
+import (
+	"math"
+	"os"
+	"path"
+	"sort"
+	"strconv"
+	"strings"
+	"sync"
+
+	logger "github.com/ElrondNetwork/elrond-go-logger"
+	"github.com/ElrondNetwork/elrond-go/core"
+	"github.com/ElrondNetwork/elrond-go/core/check"
+	"github.com/ElrondNetwork/elrond-go/data"
+	"github.com/ElrondNetwork/elrond-go/epochStart/notifier"
+	"github.com/ElrondNetwork/elrond-go/storage"
+	"github.com/ElrondNetwork/elrond-go/storage/factory/directoryhandler"
+)
+
+var log = logger.GetOrCreate("storage/olddbclean")
+
+// Args holds the arguments needed for creating an oldDatabaseCleaner
+type Args struct {
+	DatabasePath        string
+	StorageListProvider StorageListProviderHandler
+	EpochStartNotifier  EpochStartNotifier
+}
+
+type oldDatabaseCleaner struct {
+	databasePath        string
+	storageListProvider StorageListProviderHandler
+	fileRemover         func(file string) error
```

Comment:
this is actually a pathRemover (or directoryRemover)

Reply:
renamed